app/assets/images/default_thumbnail.svg

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="256.74667mm"
+   height="401.24445mm"
+   viewBox="0 0 909.73231 1421.7323"
+   id="svg4144"
+   version="1.1"
+   inkscape:version="0.91 r13725"
+   sodipodi:docname="Mobile_phone_font_awesome.svg">
+  <defs
+     id="defs4146" />
+  <sodipodi:namedview
+     id="base"
+     pagecolor="#ffffff"
+     bordercolor="#666666"
+     borderopacity="1.0"
+     inkscape:pageopacity="0.0"
+     inkscape:pageshadow="2"
+     inkscape:zoom="0.35"
+     inkscape:cx="-28.705287"
+     inkscape:cy="676.58044"
+     inkscape:document-units="px"
+     inkscape:current-layer="layer1"
+     showgrid="false"
+     fit-margin-top="20"
+     fit-margin-left="20"
+     fit-margin-right="20"
+     fit-margin-bottom="20"
+     inkscape:window-width="1317"
+     inkscape:window-height="744"
+     inkscape:window-x="49"
+     inkscape:window-y="24"
+     inkscape:window-maximized="1" />
+  <metadata
+     id="metadata4149">
+    <rdf:RDF>
+      <cc:Work
+         rdf:about="">
+        <dc:format>image/svg+xml</dc:format>
+        <dc:type
+           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
+        <dc:title></dc:title>
+      </cc:Work>
+    </rdf:RDF>
+  </metadata>
+  <g
+     inkscape:label="Capa 1"
+     inkscape:groupmode="layer"
+     id="layer1"
+     transform="translate(74.866142,212.78964)">
+    <g
+       transform="matrix(1,0,0,-1,-4,1138.0765)"
+       id="g3003">
+      <path
+         d="m 464,128 q 0,33 -23.5,56.5 Q 417,208 384,208 351,208 327.5,184.5 304,161 304,128 304,95 327.5,71.5 351,48 384,48 417,48 440.5,71.5 464,95 464,128 Z m 208,160 0,704 q 0,13 -9.5,22.5 -9.5,9.5 -22.5,9.5 l -512,0 q -13,0 -22.5,-9.5 Q 96,1005 96,992 l 0,-704 q 0,-13 9.5,-22.5 Q 115,256 128,256 l 512,0 q 13,0 22.5,9.5 9.5,9.5 9.5,22.5 z m -192,848 q 0,16 -16,16 l -160,0 q -16,0 -16,-16 0,-16 16,-16 l 160,0 q 16,0 16,16 z m 288,16 0,-1024 Q 768,76 730,38 692,0 640,0 L 128,0 Q 76,0 38,38 0,76 0,128 l 0,1024 q 0,52 38,90 38,38 90,38 l 512,0 q 52,0 90,-38 38,-38 38,-90 z"
+         id="path3005"
+         inkscape:connector-curvature="0"
+         style="fill:currentColor" />
+    </g>
+  </g>
+</svg>
```

Comment:
This file isn't used, right?

Reply:
I intended to use it if I know how to play with SVG, but it's ok for me to delete it as I don't need it atm.